Content.Server/Actions/Events/DisarmAttemptEvent.cs

```diff
@@ -0,0 +1,17 @@
+using Robust.Shared.GameObjects;
+
+
+namespace Content.Server.Actions.Events
+{
+    public class DisarmAttemptEvent : CancellableEntityEventArgs
+    {
+        public readonly EntityUid TargetUid;
+        public readonly EntityUid? DisarmerUid;
```

Comment:
why is this nullable. shouldnt it always  exist

Reply:
Theoretically there could be an effect from something that tries to disarm